### PR TITLE
add plant-cards

### PR DESCRIPTION
**Pull request by gulfemakkaya**
My PR for my  US #1

### CODE REVIEW

**Review by IrisHerholz — APPROVED** [review]
Hey Gülfem, your code looks solid to me :)

File: components/PlantCard/index.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+                <h2>{plant.name}</h2>
+                <br />
+
+                {
+                  <Image
+                    src={plant.image}
+                    width={wdt}
+                    height={hdt}
+                    alt={plant.name}
+                  />
```
Comment (AstridOttmann) [inline components/PlantCard/index.js:82]: Hey Gülfem :-)
you don't have to declare const for width and height, you can put it directly in: width={100}

Comment (gulfemakkaya) [reply components/PlantCard/index.js:80]: Hey Astrid :) Thanks for your review changed it!

File: components/PlantCard/index.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      image:
+        "https://images.pexels.com/photos/993018/pexels-photo-993018.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2",
+    },
+    {
+      name: "Philodendron",
+      image:
+        "https://images.pexels.com/photos/7663987/pexels-photo-7663987.jpeg",
+    },
+  ]);
+
```
Comment (AstridOttmann) [inline components/PlantCard/index.js:61]: To have your data-object as initial value in the useState is correct, but it doesn't look very nice. I would suggest to store your plants in a const and create a separate file for the data. Then. pass the const to the useState.

File: components/PlantCard/index.js (hunk 1 above)
Comment (AstridOttmann) [reply components/PlantCard/index.js:80]: Please remove the console-log in line 66 (when starting the PR all console-logs should be deleted, but I forget it very often too ;-)
Do you need the <br /> in line 72? If you want to create more space between elements it is good practice to use styling.

File: components/PlantCard/index.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    </>
+  );
+}
+
+const ListItem = styled.li`
+  border: 2px solid grey;
+  border-radius: 10px 10px;
+  margin: 10px 10px 10px 10px;
+  text-align: center;
+`;
```
Comment (AstridOttmann) [inline components/PlantCard/index.js:96]: If your margin is the same for all you can use:
margin: 10px;
Same for the border-radius

File: components/PlantCard/index.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+const StyledList = styled.ul`
+  list-style-type: none;
+  margin: 2px 2px 2px 2px;
+  padding: 2px 2px 2px 2px;
+`;
```
Comment (AstridOttmann) [inline components/PlantCard/index.js:102]: And the padding ;-)

File: pages/index.js [hunk 5]
```diff
@@ -1,3 +1,15 @@
...
-  return <h1>Hello from Next.js</h1>;
+  return (
+    <>
+      <main>
+        <TitleBar />
+        <PlantCard />
+      </main>
+    </>
+  );
 }
```
Comment (AstridOttmann) [inline pages/index.js:15]: You can think about moving the header to the _app later on, I suppose you want to have it on each side. 
The structure of the body is: 
header
main
footer (if you have one)

Comment (AstridOttmann) [reply pages/index.js:15]: In your US you write "the title is always visible".
That is not the case: 
You can fix this with:  
position: fixed; 
the parent element needs position: relative;

Comment (AstridOttmann) [reply pages/index.js:15]: It's a really good start! 
Most of the things I commented are only small things.
Ask me if you have further questions:-)

**Pull request merged**